src/notebook_utils/notebook_utils.py

```diff
@@ -0,0 +1,41 @@
+import itables.options as opt
+from itables import init_notebook_mode, show
+from pandas import DataFrame
+
+
+def display_df(
+        df: DataFrame,
+        layout: dict = None,
+        buttons: list = None,
+        length_menu: list = None
+) -> None:
+    """
+    Display a pandas DataFrame using itables.
+    iTables project page: https://github.com/mwouts/itables
+
+    :param df: a pandas DataFrame
+    :param layout: layout options, refer to https://datatables.net/reference/option/layout
+    :param buttons: buttons options, options refer to https://datatables.net/reference/button/
+    :param length_menu: length menu options
```

Comment:
I have no idea what length menu does based on reading the docs and the code

Reply:
👍 added a reference.